registry/auth/token/token.go

```diff
@@ -20,6 +20,9 @@ const (
 	// TokenSeparator is the value which separates the header, claims, and
 	// signature in the compact serialization of a JSON Web Token.
 	TokenSeparator = "."
+	// NotBeforeLeeway is the value in seconds that will be added to the NBF claim
+	// check to account for clock skew as per https://tools.ietf.org/html/rfc7519#section-4.1.5
+	NotBeforeLeeway = 60
```

Comment:
That's fine but we're dealing int64 types on the checks so we will need to cast the duration type to seconds.